lib/spoom/coverage.rb

```diff
@@ -0,0 +1,72 @@
+# typed: strict
+# frozen_string_literal: true
+
+require_relative "coverage/snapshot"
+require_relative "coverage/report"
+require_relative "file_tree"
+
+require "date"
+
+module Spoom
+  module Coverage
+    extend T::Sig
+
+    sig { params(path: String).returns(Snapshot) }
+    def self.snapshot(path: '.')
+      snapshot = Snapshot.new
+      metrics = Spoom::Sorbet.srb_metrics(path: path, capture_err: true)
+      return snapshot unless metrics
+
+      sha = Spoom::Git.last_commit(path: path)
+      snapshot.commit_sha = sha
+      snapshot.commit_timestamp = Spoom::Git.commit_timestamp(sha, path: path).to_i if sha
```

Comment:
I guess my previous comment is invalid now and the question is this `sha` can be null 😲